### PR TITLE
Update to latest Welsh translations

### PR DESCRIPTION
**Pull request by ajmaddaford**
### What is the context of this PR?
Update to the latest Welsh translations from Crowdin.

### How to review 
Check that the translations are up-to-date.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by rmccar — APPROVED** [review]

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -689,583 +682,499 @@ msgstr "Dileu"
...
+#: templates/static/accessibility.html:103
+msgid "Over the last few years we have tested the full questionnaire, including:"
+msgstr "Dros y blynyddoedd diwethaf, rydym wedi profi'r holiadur llawn, gan gynnwys:"
 
-#: templates/static/cookies-privacy.html:327
-msgid "Survey Data Collection captures and stores the email address of those who register with the platform. Registration is required in order to complete a survey return."
-msgstr "Mae Casglu Data'r Arolwg yn casglu ac yn storio cyfeiriad e-bost y rhai sy'n cofrestru gyda'r llwyfan. Mae angen cofrestru er mwyn cwblhau arolwg."
+#: templates/static/accessibility.html:106
+msgid "<a href='{url}'>Digital Accessibility Centre (DAC)</a> carrying out a professional, technical audit"
+msgstr "●\tCael y <a href='{url}'>Ganolfan Hygyrchedd Digidol (DAC)</a> i gynnal archwiliad technegol proffesiynol"
```
Comment (MebinAbraham) [inline app/translations/cy/LC_MESSAGES/messages.po]: ●\t?

File: app/translations/cy/census_individual_gb_wls_cy.po [hunk 2]
```diff
@@ -196,7 +195,7 @@ msgid "Include 4 or more AS levels"
 msgstr "Dylech gynnwys 4 neu fwy o gymwysterau Lefel AS (Safon UG)"
 
 msgid "Include 2 to 3 AS levels"
-msgstr "Dylech gynnwys 2-3 Lefel AS (Safon UG)"
+msgstr "Dylech gynnwys 2–3 Lefel AS (Safon UG)"
```
Comment (JamesGardiner) [inline app/translations/cy/census_individual_gb_wls_cy.po:198]: Is this valid translation? @berroar is there a better way to represent 2 to 3 in Welsh?

File: app/translations/cy/census_individual_gb_wls_cy.po [hunk 3]
```diff
@@ -754,10 +753,10 @@ msgid "What was <em>{person_name_possessive}</em> full job title?"
 msgstr "Beth oedd teitl llawn swydd <em>{person_name_possessive}</em>?"
 
 msgid "Briefly describe what <em>{person_name}</em> does in their main job."
-msgstr "Beth mae <em>{person_name}</em> yn ei wneud yn y brif swydd."
+msgstr "Beth mae <em>{person_name}</em> yn ei wneud yn y brif swydd?"
 
 msgid "Briefly describe what <em>{person_name}</em> did in their main job."
-msgstr "Beth oedd <em>{person_name}</em> yn ei wneud yn y brif swydd."
+msgstr "Beth oedd  yn ei wneud yn y brif swydd?"
```
Comment (JamesGardiner) [inline app/translations/cy/census_individual_gb_wls_cy.po]: This is missing the placeholder...

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -689,583 +682,499 @@ msgstr "Dileu"
...
+#: templates/static/privacy.html:68
+msgid "processing is necessary for archiving in the public interest, scientific or historical research purposes or statistical purposes based on UK law."
+msgstr "•\tmae prosesu yn angenrheidiol ar gyfer archifo er budd y cyhoedd, at ddibenion ymchwil wyddonol neu hanesyddol neu at ddibenion ystadegol yn seiliedig ar gyfraith y DU."
 
-#: templates/static/cookies-privacy.html:465
-msgid "Standard logs from the Survey Data Collection service are retained for a period of 30 days. Exceptional events may be retained for longer."
-msgstr "Cedwir cofnodion sylfaenol gan wasanaeth Casglu Data'r Arolwg am gyfnod o 30 niwrnod. Gellir cadw digwyddiadau arbennig am gyfnod hwy."
+#: templates/static/privacy.html:69 templates/static/privacy.html:75
+msgid "processing is necessary for reasons of substantial public interest."
+msgstr "•\tmae prosesu yn angenrheidiol ar sail budd sylweddol i'r cyhoedd."
```
Comment (MebinAbraham) [inline app/translations/cy/LC_MESSAGES/messages.po]: •\t Should we be allowing this?

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: app/translations/cy/census_individual_gb_wls_cy.po (hunk 3 above)
Comment (JamesGardiner) [reply app/translations/cy/census_individual_gb_wls_cy.po]: Also that space between `oedd` and `yn` is actually two spaces

File: app/translations/cy/LC_MESSAGES/messages.po (hunk 1 above)
Comment (ajmaddaford) [reply app/translations/cy/LC_MESSAGES/messages.po]: Removed

File: app/translations/cy/LC_MESSAGES/messages.po (hunk 4 above)
Comment (ajmaddaford) [reply app/translations/cy/LC_MESSAGES/messages.po]: Removed

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -400,37 +411,37 @@ msgstr "Byddwch yn cael y cyfle i weld ac argraffu copi o'ch atebion ar ôl cyfl
...
+#: templates/view-submission.html:19
 #, python-format
 msgid "Your answers were submitted for <span>%(ru_name)s</span> (%(trading_as_name)s) on %(submitted_time)s"
 msgstr "Cyflwynwyd eich atebion ar gyfer <span>%(ru_name)s</span> (%(trading_as_name)s) ar %(submitted_time)s"
 
-#: templates/view-submission.html:25
+#: templates/view-submission.html:24
 #, python-format
 msgid "Your answers were submitted for <span>%(ru_name)s</span> on %(submitted_time)s"
 msgstr "Cyflwynwyd eich atebion ar gyfer  <span>%(ru_name)s</span> ar %(submitted_time)s"
```
Comment (MebinAbraham) [inline app/translations/cy/LC_MESSAGES/messages.po:438]: I know this is existing but while you're at it?
```suggestion
msgstr "Cyflwynwyd eich atebion ar gyfer <span>%(ru_name)s</span> ar %(submitted_time)s"
```

File: app/translations/cy/census_individual_gb_wls_cy.po (hunk 3 above)
Comment (ajmaddaford) [reply app/translations/cy/census_individual_gb_wls_cy.po]: Fixed

File: app/translations/cy/census_individual_gb_wls_cy.po (hunk 2 above)
Comment (ajmaddaford) [reply app/translations/cy/census_individual_gb_wls_cy.po:198]: Rhys said it's fine

Comment (ajmaddaford) [reply app/translations/cy/census_individual_gb_wls_cy.po:198]: Rhys said it's fine

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -889,7 +889,7 @@ msgstr "Dros y blynyddoedd diwethaf, rydym wedi profi'r holiadur llawn, gan gynn
 
 #: templates/static/accessibility.html:106
 msgid "<a href='{url}'>Digital Accessibility Centre (DAC)</a> carrying out a professional, technical audit"
-msgstr "●\tCael y <a href='{url}'>Ganolfan Hygyrchedd Digidol (DAC)</a> i gynnal archwiliad technegol proffesiynol"
+msgstr "cael y <a href='{url}'>Ganolfan Hygyrchedd Digidol (DAC)</a> i gynnal archwiliad technegol proffesiynol"
```
Comment (JamesGardiner) [inline app/translations/cy/LC_MESSAGES/messages.po:892]: Should `cael` be `Cael`?

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: app/translations/cy/LC_MESSAGES/messages.po (hunk 5 above)
Comment (ajmaddaford) [reply app/translations/cy/LC_MESSAGES/messages.po:438]: Assuming the build passes we'll need to leave this as the latest Welsh translations are needed for testing imminently.

File: app/translations/cy/LC_MESSAGES/messages.po (hunk 6 above)
Comment (ajmaddaford) [reply app/translations/cy/LC_MESSAGES/messages.po:892]: no, it's a bullet point and all other bullet points start with a lowercase character.

Comment (JamesGardiner) [reply app/translations/cy/LC_MESSAGES/messages.po:892]: :+1:

**Review by MebinAbraham — APPROVED** [review]

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**